ViewModel/HyvaGridViewModel.php

```diff
@@ -89,9 +94,19 @@ private function buildColumnDefinitions(): array
         $keysToHide       = $this->getGridDefinition()->getExcludedColumnKeys();
         $availableColumns = $this->getGridSourceModel()->extractColumnDefinitions($columns, $keysToHide, $showAll);
 
+        $availableColumns = $this->preprocessColumnDefinitions($availableColumns);
```

Comment:
This statement overwrites the variable `$availableColums` with the return value from `preprocessColumnDefinitions`.
I try to avoid re-assigning already declared variables.
This helps me write less buggy code, especially over time when I revisit a method I haven't worked on for a while.

Can you please rename the result to something else, for example `$preprocessedColumns`?